EpsilonDriverControls/Src/DriverControls.c

```diff
@@ -100,6 +102,21 @@ void sendDriverTask(void const* arg)
         hcan2.pTxMsg->Data[1] = 0;
         hcan2.pTxMsg->Data[2] = 0;
         hcan2.pTxMsg->Data[3] = 0;
+
+        if (HAL_ADC_PollForConversion(&hadc1, 1000000) == HAL_OK)
```

Comment:
So if it is unable to read, so send last value? I am unsure of that with the safety implications of having an input stuck high. But setting 0 could be lots of change of some read and some don't but overall I think it would be safer (ideally set 0 into running average).

I think this might need a comment highlighting what is happening here.